portal/src/views/shared/SidebarNav.vue

```diff
@@ -60,6 +60,7 @@
 
 <script lang="ts">
 import Vue from "vue";
+import EventBus from 'vue';
```

Comment:
I believe this is being deprecated. No big deal cause it doesn't look like it's used?